cli/tests/integration_tests.rs

```diff
@@ -5138,6 +5138,25 @@ console.log("finish");
     assert!(stderr.contains("BadResource"));
   }
 
+  #[cfg(not(windows))]
```

Comment:
The bug the PR is fixing doesn't seem to be a bug on windows as per https://github.com/denoland/deno/runs/2373907680?check_suite_focus=true#step:28:2087 hence not running the test on windows.